notebooks/example.ipynb

```diff
@@ -839,22 +839,68 @@
    drs_uri: drs://dg.4DFC:4e3f5068-6b5e-11e9-9244-005056921935\n\n    byte_size: 5299987\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f10.mzid.gz\n \n    drs_uri: drs://dg.4DFC:4e8aaeb4-6b5e-11e9-9244-005056921935\n\n    byte_size: 5451969\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f11.mzid.gz\n \n    drs_uri: drs://dg.4DFC:4ee07a60-6b5e-11e9-9244-005056921935\n\n    byte_size: 5637958\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f04.mzid.gz\n \n    drs_uri: drs://dg.4DFC:4f319260-6b5e-11e9-9244-005056921935\n\n    byte_size: 5514135\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f03.mzid.gz\n \n    drs_uri: drs://dg.4DFC:4faaa038-6b5e-11e9-9244-005056921935\n\n    byte_size: 4930932\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f02.mzid.gz\n \n    drs_uri: drs://dg.4DFC:4ff94efe-6b5e-11e9-9244-005056921935\n\n    byte_size: 4949776\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f08.mzid.gz\n \n    drs_uri: drs://dg.4DFC:5073190a-6b5e-11e9-9244-005056921935\n\n    byte_size: 5955432\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f12.mzid.gz\n \n    drs_uri: drs://dg.4DFC:510107ba-6b5e-11e9-9244-005056921935\n\n    byte_size: 5189966\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_W_BI_20130222_H-PM_f05.raw\n \n    drs_uri: drs://dg.4DFC:526bfee2-5cd2-11e9-849f-005056921935\n\n    byte_size: 1305560173\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f09.mzid.gz\n \n    drs_uri: drs://dg.4DFC:515c58ea-6b5e-11e9-9244-005056921935\n\n    byte_size: 5806522\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f07.mzid.gz\n \n    drs_uri: drs://dg.4DFC:51b5d24e-6b5e-11e9-9244-005056921935\n\n    byte_size: 4640215\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_fA.mzid.gz\n \n    drs_uri: drs://dg.4DFC:51e70058-6b5e-11e9-9244-005056921935\n\n    byte_size: 1674176\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_P_BI_20130228_H-PM_f05.mzid.gz\n \n    drs_uri: drs://dg.4DFC:52457d2c-6b5e-11e9-9244-005056921935\n\n    byte_size: 5685188\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_W_BI_20130222_H-PM_f06.raw\n \n    drs_uri: drs://dg.4DFC:5b0c55c4-5cd2-11e9-849f-005056921935\n\n    byte_size: 1280644725\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_W_BI_20130222_H-PM_f07.raw\n \n    drs_uri: drs://dg.4DFC:634a6b40-5cd2-11e9-849f-005056921935\n\n    byte_size: 1341011263\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_W_BI_20130222_H-PM_f01.raw.cap.psm\n \n    drs_uri: drs://dg.4DFC:6bc4589c-6b5c-11e9-ab5c-005056921935\n\n    byte_size: 6923743\n\n    \n\n    label: TCGA_E2-A10A_BH-A18Q_C8-A130_117C_W_BI_20130222_H-PM_f02.raw.cap.psm\n \n    drs_uri: drs://dg.4DFC:6bf22560-6b5c-11e9-ab5c-005056921935\n\n    byte_size: 6847471\n\n    \n"
```

Comment:
Phil suggested that we avoid changing the state of the JupyterNotebooks unless we are actually updating the functionality.   I am thinking that we should put the *.ipynb into the .gitignore file for the most part unless we are checking in a new notebook.